src/main/kotlin/com/sourcegraph/cody/config/migration/SettingsMigration.kt

```diff
@@ -61,16 +60,22 @@ class SettingsMigration : Activity {
     RunOnceUtil.runOnceForProject(project, "CodyAssignOrphanedChatsToActiveAccount") {
       migrateOrphanedChatsToActiveAccount(project)
     }
-
-    DeprecatedChatLlmMigration.migrate(project)
-    ChatTagsLlmMigration.migrate(project)
+    RunOnceUtil.runOnceForProject(project, "DeprecatedChatLlmMigration") {
+      DeprecatedChatLlmMigration.migrate(project)
+    }
+    RunOnceUtil.runOnceForProject(project, "ChatTagsLlmMigration") {
+      ChatTagsLlmMigration.migrate(project)
+    }
     RunOnceUtil.runOnceForProject(project, "CodyMigrateChatHistory-v2") {
       ChatHistoryMigration.migrate(project)
     }
+    RunOnceUtil.runOnceForProject(project, "AccountsToCodyMigration") {
+      AccountsMigration.migrate()
```

Comment:
Just curious but are these guaranteed to run *in this order*? Is it important?

Reply:
AFAIR their are, this code is synchronous.